Firebase/Auth/Source/Auth/FIRAuth.m

```diff
@@ -1136,10 +1136,17 @@ - (void)sendSignInLinkToEmail:(nonnull NSString *)email
           kMissingEmailInvalidParameterExceptionReason];
     }
 
+#if TARGET_OS_WATCH
```

Comment:
Nit: apologies I think I messed up on this one comment - can you please keep this one as `!TARGET_OS_WATCH` (since we explicitly want to exclude watchOS and don't have any alternative code to run for it) and move the comment to above the `#if !TARGET_OS_WATCH`?